.changeset/long-ants-sing.md

```diff
@@ -0,0 +1,9 @@
+---
+'@sl-design-system/radio-group': patch
```

Comment:
I don't see any changes to these components?

Reply:
not to the components, but i did change the story. That doesn't warrant a mention in the changeset?

Reply:
No, the changelog is only for API related things (in this case the web components we ship).